environment.yml

```diff
@@ -39,8 +39,9 @@ dependencies:
         - sacremoses
 
         # Warning: jiant currently depends on *both* pytorch_pretrained_bert > 0.6 _and_
```

Comment:
Check if this is still true—is the old package still a dependency via Allen? If not, delete.

Reply:
allennlp v0.8.4 still depends on pytorch-pretrained-bert, and we've not changed the allennlp requirement.

Reply:
Dang. I guess it's not worth the effort to update the Allen dependency, assuming that the 2.0 migration is coming up fairly soon.